README.md

```diff
@@ -48,19 +49,19 @@ you can omit this parameter.**
 
 Usage
 ---------------------
-To build simple [nodejs-echo-app](https://github.com/ryanj/node-echo) application,
+To build simple [nodejs-sample-app](https://github.com/openshift/sti-nodejs/tree/master/0.10/test/test-app) application,
 using standalone [STI](https://github.com/openshift/source-to-image) and then run the
 resulting image with [Docker](http://docker.io) execute:
 
 *  **For RHEL based image**
     ```
-    $ sti build https://github.com/ryanj/node-echo.git openshift/nodejs-010-rhel7 nodejs-echo-app
+    $ sti build https://github.com/openshift/sti-nodejs.git --contextDir=/0.10/test/test-app/ openshift/nodejs-010-rhel7 nodejs-sample-app
```

Comment:
why you care? you just want to build the application with this image, you don't care about all the ugly options to the sti command. The example will be then:

```
$ cd repo
$ sti build --use-config
$ docker run
```